### PR TITLE
Added info about jo and slides with dates

### PR DESCRIPTION
**Pull request by joph**
Just to see if I can do a proper pull request... I added info about me and a slide with dates.

### CODE REVIEW

File: lecture00-introduction/introduction.tex [hunk 1]
```diff
@@ -153,7 +153,9 @@ \section{Who we are}
 \begin{frame}{Who we are}
     Johannes Schmidt <johannes.schmidt@boku.ac.at>\\
     \begin{itemize}
-        \item
+        \item Associate Prof. in Energy \& Resource Economics at the Institute for Sustainable Economic Development
+        \item Works on modeling of renewable energy systems in R, GAMS (and python)
```
Comment (lumbric) [inline lecture00-introduction/introduction.tex]: Python is usually spelled "Python" AFAIK

File: lecture00-introduction/introduction.tex [hunk 2]
```diff
@@ -153,7 +153,9 @@ \section{Who we are}
...
+        \item Has studied Computer Science at TU Wien
```
Comment (lumbric) [inline lecture00-introduction/introduction.tex]: I think we should agree on English grammar if possible, is it "Has studied" or "Studied" in this case?

**Review by lumbric — CHANGES_REQUESTED** [review]

File: lecture00-introduction/introduction.tex (hunk 1 above)
Comment (joph) [reply lecture00-introduction/introduction.tex]:
```suggestion
        \item Works on modeling of renewable energy systems in R, GAMS (and Python)
```

File: lecture00-introduction/introduction.tex (hunk 2 above)
Comment (joph) [reply lecture00-introduction/introduction.tex]:
```suggestion
        \item Studied Computer Science at TU Wien
```

**Comment by lumbric** [comment]
I think there should be a button, where you can re-request a review (to state that you are done with updating the pull-request according to my request). But let's skip this for now and merge it!

Looks good now, thanks for the changes!

**Pull request merged**